PA1PartTwo.asm

```diff
@@ -0,0 +1,85 @@
+# COMP3410 Program Template
+# Author: Robert L. Edstrom
+# Assignment: PA[1]
+# Date: 
+
+# PA1PartTwo.asm
+# Input: two integers
+# Output: introduction along with the sum, difference, and product of two integers
+	
+	.data
+
+intro:	.asciiz	"Welcome, I am Robert Edstrom and this is PA1 part-two. \nInput: two integers, Output: sum, difference, & product\n"
+msg0:	.asciiz	"Pick an integer: "
+add2:	.asciiz	"Sum: "
+sub2:	.asciiz	"\nDifference: "
+mult2: 	.asciiz "\nProduct: "
+
+	.text
+		
+		# Program setup
+		la  $a0, intro		# Load the address of intro into $a0
+		li  $v0, 4		# Setup for printing the intro
+		syscall			# Syscall to print the intro
+		
+		la  $a0, msg0		# Load the address of msg0 into $a0
+		li  $v0, 4		# Setup for printing msg0
+		syscall 		# Syscall to print msg0 
+		
+		li  $v0, 5		# Setup for accepting an integer as user input
+		syscall			# Syscall to prompt the user for an integer
+		move $t0, $v0		# Move the integer from $v0 to $t0
+		
+		la  $a0, msg0		# Load the address of msg0 into $a0
+		li  $v0, 4		# Setup for printing msg0 
+		syscall			# Syscall to print msg0
+		
+		li  $v0, 5		# Setup for accepting a second integer as user input
+		syscall			# Syscall to prompt the user for an integer
+		move $t1, $v0		# Move the second integer from $v0 to $t0
+		# End program setup
+		
+		jal addInts		# Jump to addInts to perform addition on the two integers
+		jal subInts		# Jump to subInts to perform subtraction on the two integers
+		jal multInts		# Jump to multInts to perfrom multiplication on the two integers
+		
+		li $v0, 10		# Setup for system exit
+		syscall			# Syscall to exit
+		
+		
+addInts: 	add $t2, $t0, $t1	# Add values stored in $t0 and $t1 and store the result in $t2
+		la  $a0, add2		# Load address of add2 message into $a0
+		li  $v0, 4		# Setup for printing add2 message
+		syscall			# Syscall to print add2 message
+		
```

Comment:
-8 Instructions require that you store (-4) arithmetic results to memory and retrieve (-4) them from memory for printing